node/src/utils.rs

```diff
@@ -161,7 +162,7 @@ pub fn read_file_to_string<P: AsRef<Path>>(filename: P) -> Result<String, ReadFi
 /// With-directory context.
 ///
 /// Associates a type with a "working directory".
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Deserialize, Serialize)]
```

Comment:
I don't think we need this change, do we?

Reply:
Yep we do not, left over imports from a failed iteration